tests/Containerfile-build

```diff
@@ -0,0 +1,13 @@
+FROM quay.io/centos/centos:stream9
+
+ARG USERNAME=test-user
+
+RUN dnf install -y dnf-plugin-config-manager
+RUN dnf config-manager -y --set-enabled crb && \
+    dnf install -y clang-tools-extra gcc make \
+    meson systemd-devel systemd
+RUN dnf -y clean all
```

Comment:
I think it's better to run `dnf clean all` in the same layer that you run the install. AFAIK in the current way, you're not saving any storage

Reply:
Ack,
Note: it went from 1.37G -> 1.01G